Makefile

```diff
@@ -948,3 +948,4 @@ cpu-benchmark-autopause: ## run the cpu usage auto-pause benchmark
 update-gopogh-version: ## update gopogh version
 	(cd hack/update/gopogh_version && \
 	 go run update_gopogh_version.go)
+
```

Comment:
remove unrelated

also please squash the commits, for a cleaner commmit history